packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -209,6 +209,13 @@ backbone = keras_nlp.models.Backbone.from_preset("hf://${model.id}")
 `,
 ];
 
+export const mars5_tts = (model: ModelData): string[] => [
+	`# Install from https://github.com/Camb-ai/MARS5-TTS
+
+from inference import Mars5TTS
+mars5 = Mars5TTS.from_pretrained("${model.id}")`,
```

Comment:
Lmk if this looks okay to you @osanseviero - this is good to merge from my side. 🤗